server/routerlicious/kubernetes/README.md

```diff
@@ -50,8 +50,10 @@ You'll also need to have a Redis, MongoDB, Rabbitmq, and Historian instances run
 
 We install MongoDB and Rabbitmq from the helm stable repository. We also configure MongoDB to use the managed-premium storage class in AKS.
 
-`helm install --set persistence.storageClass=managed-premium,persistence.size=4094Gi,usePassword=false stable/mongodb`
-`helm install --set rbacEnabled=false,rabbitmq.username=prague,rabbitmq.password=[rabbitmq password],persistence.enabled=true,persistence.size=16Gi stable/rabbitmq`
+In the following commands you can omit the optional key+value pairs to use the defaults defined in the Helm Chart. Also, replace the `<helm-release-name>` with the appropriate value.
```

Comment:
The changes in this file and in `server/routerlicious/kubernetes/nginx/README.md` are from an earlier PR from @ghoshkaj , do we know how they made it here? They're correct, but that PR was already merged, so this made me wonder if something went wrong with merges/rebases of something.

Reply:
I need to rebase as my branch was made off main. We may close this PR - moving it to draft